pyiron/sphinx/base.py

```diff
@@ -125,7 +126,7 @@ def plane_wave_cutoff(self):
         if "eCut" in self.input.sphinx.basis.keys():
             return self.input.sphinx.basis["eCut"] * RYDBERG_TO_EV
         else:
-            return self.input["EnCut"]
+            return self.input.sphinx["EnCut"]
```

Comment:
Are you sure that this is the right change?

Reply:
No, I guess I just got lost updating between input/ input.sphinx, etc. calls.